src/cc2olx/qti.py

```diff
@@ -211,6 +212,18 @@ def _create_fib_problem(self, problem_data):
 
         return problem
 
+    @staticmethod
+    def _build_fib_problem_type(problem_data):
+        """
+        Build `stringresponse` OLX type for a fill in the blank problem.
+        """
+        problem_types = ["ci"]
+
+        if problem_data["is_regexp"]:
```

Comment:
Should we use `problem_data.get("is_regexp")` here?

Reply:
No, we set `"is_regexp"` in `_parse_fib_problem` without any condition, so `"is_regexp"` will be presented in `problem_data`.